capabilities/index/index.go

```diff
@@ -0,0 +1,89 @@
+package index
+
+import (
+	"fmt"
+
+	"github.com/ipfs/go-cid"
+	"github.com/storacha/go-libstoracha/capabilities/types"
+	"github.com/storacha/go-ucanto/core/result/failure"
+	"github.com/storacha/go-ucanto/core/schema"
+	"github.com/storacha/go-ucanto/did"
+	"github.com/storacha/go-ucanto/ucan"
+	"github.com/storacha/go-ucanto/validator"
+)
+
+// SpaceDID represents a DID of a space
+type SpaceDID did.DID
```

Comment:
I don't think this type definition adds a lot of value. I would remove it, we use `did.DID` in the implementation of other capabilities, and I think consistency is more valuable.